### PR TITLE
Remove unused deps

### PR DESCRIPTION
**Pull request by Nutomic**
Using `cargo machete`

### CODE REVIEW

File: crates/api_common/Cargo.toml
Comment (dessalines) [inline crates/api_common/Cargo.toml:70]: Looks like it needs this one. After it passes feel free to merge tho.

**Review by dessalines — APPROVED** [review]

**Comment by dessalines** [comment]
Oh also I just tested this locally, and it'd be pretty easy / doable to add this to our CI. It tells you how to add an ignore to a dependency, as in the case of that failed one also.

**Comment by Nutomic** [comment]
Adding it to CI sounds like a good idea, then I will leave that to you. Besides the getrandom, also strum in db_schema and db_views_actor are wrongly removed.

**Pull request merged**
